src/shared/logBuffer.config.ts

```diff
@@ -15,19 +20,43 @@ export const logBuffer = () => {
   } else {
     localStorage.setItem('ringBufferContent', JSON.stringify(GlobalRingBuffer))
   }
-
   const originalFactory = log.methodFactory
   log.methodFactory = (methodName, logLevel, loggerName) => {
     const rawMethod = originalFactory(methodName, logLevel, loggerName)
-
     return (message) => {
-      //show only warnings and error in console and log everything else in the GlobalRingBuffer?
+      //show only warnings and error in console and log everything else in the GlobalRingBuffer
       if (methodName === 'warn' || methodName === 'error') {
         rawMethod(message)
         // eslint-disable-next-line no-console
         console.log(GlobalRingBuffer)
       }
-      GlobalRingBuffer.add([new Date().toUTCString(), message])
+      const date = new Date().toUTCString()
+      GlobalRingBuffer.add([date, message])
+      const array = GlobalRingBuffer.toArray()
+      //send buffer if in production or if send is true in development
+      if (isProd || sendLogs) {
+        if (methodName === 'error') {
+          //send the last BUFFER_LOG_SIZE messages
+          const bufferBody = array.slice(Math.max(array.length - config.BUFFER_LOG_SIZE, 0), array.length)
+
+          //get userid from localStorage
+          const storageData = localStorage.getItem('persisted_storage')
+          if (storageData) {
+            const persistedStorage = JSON.parse(storageData)
+            if (!(persistedStorage && persistedStorage.state && persistedStorage.state._user.id)) {
+              return
+            }
+
+            postBufferContent(bufferBody, persistedStorage.state._user.id).catch(() => {
+              console.log('buffer failed to send')
```

Comment:
Vermutlich eher eine Snackbar, falls es hier möglich ist. Ansonten wird es in log.error() reingeschreiben und vielleicht beim nächsten mal geschickt.

Reply:
snackbar geht nicht. wenn das backend nicht verfuegbar ist wird der buffer mit log.error zugespammt